common/web/keyboard-processor/src/keyboards/activeLayout.ts

```diff
@@ -90,7 +96,7 @@ export class ActiveKey implements LayoutKey {
   public get isPadding(): boolean {
     // Does not include 9 (class:  blank) as that may be an intentional 'catch' for misplaced
     // keystrokes.
-    return this['sp'] == 10; // Button class: hidden.
+    return this.sp == 10; // Button class: hidden.
```

Comment:
Huh.  There's a bit of discrepancy I noted while resolving this:

https://github.com/keymanapp/keyman/blob/f1d4b72da7d14f31256d0dc8c3ce1e13c1985ca8/common/web/types/src/keyman-touch-layout/keyman-touch-layout-file.ts#L62

vs

https://github.com/keymanapp/keyman/blob/f1d4b72da7d14f31256d0dc8c3ce1e13c1985ca8/common/web/keyboard-processor/src/keyboards/defaultLayouts.ts#L65-L75

Which also bumps up against:

https://github.com/keymanapp/keyman/blob/f1d4b72da7d14f31256d0dc8c3ce1e13c1985ca8/web/src/engine/osk/src/buttonClassNames.ts#L2-L17

I'll be using the second one, since it's the closest definition available and includes entries not seen in the `common/web/types` version.  Though... before these changes, the only other uses were within the defaultLayouts.ts file itself, most prominently within the following method:

https://github.com/keymanapp/keyman/blob/f1d4b72da7d14f31256d0dc8c3ce1e13c1985ca8/common/web/keyboard-processor/src/keyboards/defaultLayouts.ts#L409

...

This seems like something that should be common-alized as well, I suppose.  

That said, upon trying to do such a centralization... it looks as if the newer target ES environment used by `common/web/types` builds conflicts with esbuild bundling for KMW.  The problem gets triggered if I try to directly use the enum `TouchLayoutKeySp`, as it appears to mean no longer sticking strictly to `import type` mechanisms.  Non-string enums get "reverse mappings" (https://www.typescriptlang.org/docs/handbook/enums.html#reverse-mappings) that are compiled code, and I'm guessing that's enough to cause an issue here.

Reply:
See #9769 for the "better" resolution; it involves further changes to `common/web/types`, so I felt that splitting it off as a separate PR would be wisest - there may be consequences to those changes that I'm not aware of.